salt/states/cron.py

```diff
@@ -606,6 +606,7 @@ def file(name,
             user,
             group,
             mode,
+            [],
```

Comment:
I know this wasn't your call originally, but maybe we should change all of these kwargs to use their `kwarg=thing` syntax to prevent regressions in the future.

Reply:
This would be helpful.  I'll update the PR.